x-pack/packages/security-solution/ecs_data_quality_dashboard/impl/data_quality/helpers.test.ts

```diff
@@ -791,14 +791,14 @@ describe('helpers', () => {
           indexName,
           stats: mockStatsGreenIndex,
         })
-      ).toEqual(mockStatsGreenIndex[indexName].primaries?.docs?.count);
```

Comment:
The doc count of a  health`green` index was coming from `mockStatsGreenIndex[indexName].primaries?.docs?.count` where `primaries.docs.count` and `total.docs.count` have different values'